modules/cfe_testcase/src/time_arithmetic_test.c

```diff
@@ -0,0 +1,119 @@
+/*************************************************************************
+**
+**      GSC-18128-1, "Core Flight Executive Version 6.7"
+**
+**      Copyright (c) 2006-2019 United States Government as represented by
+**      the Administrator of the National Aeronautics and Space Administration.
+**      All Rights Reserved.
+**
+**      Licensed under the Apache License, Version 2.0 (the "License");
+**      you may not use this file except in compliance with the License.
+**      You may obtain a copy of the License at
+**
+**        http://www.apache.org/licenses/LICENSE-2.0
+**
+**      Unless required by applicable law or agreed to in writing, software
+**      distributed under the License is distributed on an "AS IS" BASIS,
+**      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+**      See the License for the specific language governing permissions and
+**      limitations under the License.
+**
+** File: time_arithmetic_test.c
+**
+** Purpose:
+**   Functional test of basic Time Arithmetic APIs
+**
+**   Demonstration of how to register and use the UT assert functions.
+**
+*************************************************************************/
+
+/*
+ * Includes
+ */
+
+#include "cfe_test.h"
+
+void TestTimeAdd(void)
+{
+    UtPrintf("Testing: CFE_TIME_Add");
+    CFE_TIME_SysTime_t time1     = {1000, 0};
+    CFE_TIME_SysTime_t time2     = {0, 1000};
+    CFE_TIME_SysTime_t timeAdded = CFE_TIME_Add(time1, time2);
+
+    UtAssert_UINT32_EQ(timeAdded.Seconds, 1000);
```

Comment:
Instead of checking if it equqals 1000, you should create a timeExpected var and compare to that.